mobile/native/src/db/mod.rs

```diff
@@ -0,0 +1,58 @@
+pub mod models;
+
+use crate::db::models::LastLogin;
+use anyhow::anyhow;
+use anyhow::Context;
+use anyhow::Result;
+use diesel::r2d2;
+use diesel::r2d2::ConnectionManager;
+use diesel::r2d2::Pool;
+use diesel::r2d2::PooledConnection;
+use diesel::SqliteConnection;
+use diesel_migrations::embed_migrations;
+use diesel_migrations::EmbeddedMigrations;
+use diesel_migrations::MigrationHarness;
+use state::Storage;
+use std::sync::Arc;
+use time::OffsetDateTime;
+
+pub const MIGRATIONS: EmbeddedMigrations = embed_migrations!();
+
+/// Wallet has to be managed by Rust as generics are not support by frb
```

Comment:
🙃  That comment does not make any sense.

Reply:
lol 🙈